packages/core/src/stateUtils.ts

```diff
@@ -1485,15 +1521,26 @@ interface BuiltinAction {
   execute: (actorScope: AnyActorScope, params: unknown) => void;
 }
 
-export let executingCustomAction:
-  | ActionFunction<any, any, any, any, any, any, any, any, any>
-  | false = false;
+export interface ExecutableAction {
+  /** The action type */
+  type: string;
+  info: ActionArgs<MachineContext, EventObject, EventObject>;
+  params: NonReducibleUnknown;
+  function:
```

Comment:
nit: `function` used as a property name looks weird ;p `exec`/`execute` would fit my brain better

Reply:
Renamed to `exec`